docs/design/design-patterns/non-functional-requirements-investigation-guide.md

```diff
@@ -0,0 +1,148 @@
+# Non-Functional Requirements Investigation
```

Comment:
Nit: Suggest "Non-Functional Requirements Capture" guide as we should consider positioning this as a guide to help people capture non-functional requirements in a more consistent way.

Reply:
Done,